docs/appendix/upgrade-notes/3.x-to-4.0.rst

```diff
@@ -728,12 +734,17 @@ Lib functions
 * ``diagnostics_md5_dir()``
 * ``elgg_add_subscription()`` use ``\ElggEntity::addSubscription()``
 * ``elgg_get_available_languages()`` use ``elgg()->translator->getAvailableLanguages()``
+* ``elgg_get_all_plugin_user_settings()`` use ``$plugin->getAllUserSettings($$user_guid)``
+* ``elgg_get_entities_from_plugin_user_settings()``
```

Comment:
are you sure you wish to do this. Then please offer some way for plugin devs to get the correct private setting prefix